styleguide/style.md

```diff
@@ -0,0 +1,471 @@
+
+Second-level headings will generate the on-page TOC that appears in the "In this article" section underneath the on-page title.
+
+### Third-level heading
+#### Fourth-level heading
+
+## Text styling
+
+**Bold**
+Use for UI elements.
+
+`Code`
+Use for inline code, language keywords, package names, command-line commands, database table and column names, and URLs that you don't want to be clickable.
+Use for files, folders, paths (for long items, split onto their own line), new terms.
+
+### Special characters
+
+Markdown uses special characters such as \*, \`, and \# for formatting. If you wish to include one of these characters in your content, you must do one of two things:
+
+- Put a backslash before the special character to "escape" it (for example, `\*` for a \*)
+- Use the [HTML entity code](http://www.ascii.cl/htmlcodes.htm) for the character (for example, `&#42;` for a &#42;).
+
+
+## Lists
+
+### Ordered lists
+
+1. This
+2. Is
+3. An
+4. Ordered
+5. List
+
+### Unordered lists
+
+- This
+- is
+- a
+- bulleted
+- list
+
+
+### Nested list spacing
+
+When nesting lists, use a 4 space indent for both numbered and bulleted lists:
+
+```markdown
+1.  2 spaces after a numbered list.
+    4 space indent for wrapped text.
+2.  2 spaces again.
+
+-   3 spaces after a bullet.
+    4 space indent for wrapped text.
+    1.  2 spaces after a numbered list.
+        8 space indent for the wrapped text of a nested list.
+    2.  Looks nice, don't it?
+-   3 spaces after a bullet.
+```
+
+The following works, but it's very messy:
+
+```markdown
+- One space,
+with no indent for wrapped text.
+     1. Irregular nesting... DO NOT DO THIS.
+```
+
+Even when there's no nesting, using the 4 space indent makes layout consistent
+for wrapped text:
+
+```markdown
+-   Foo,
+    wrapped.
+
+1.  2 spaces
+    and 4 space indenting.
+2.  2 spaces again.
+```
+
+However, when lists are small, not nested, and a single line, one space can suffice for both kinds of lists:
+
+```markdown
+- Foo
+- Bar
+- Baz.
+
+1. Foo.
+2. Bar.
+```
+
+## Links
+
+### Internal Links
+
+To link to a header in the same Markdown file (also known as anchor links), you'll need to find out the id of the header you're trying to link to. To confirm the ID, view the source of the rendered article, find the id of the header (for example, `id="blockquote"`), and link using # + id (for example, `#blockquote`).
+The id is auto-generated based on the header text. So, for example, given a unique section named `## Step 2`, the id would look like this `id="step-2"`.
+
+- For example: [Chapter 1](#chapter-1)
+
+To link to a Markdown file in the same repo, use [relative links](https://www.w3.org/TR/WD-html40-970917/htmlweb.html#h-5.1.2), including the ".md" at the end of the filename.
+
+- For example: [Readme file](../readme.md)
```

Comment:
Is it just a sample? This link is not working.

Reply:
done.